packages/components/src/organisms/Newsletter/NewsletterAddendum.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { forwardRef } from 'react'
+import type { HTMLAttributes, DetailedHTMLProps } from 'react'
+
+export interface NewsletterAddendumProps
+  extends DetailedHTMLProps<HTMLAttributes<HTMLSpanElement>, HTMLSpanElement> {
+  /**
+   * Expects a string of a JSON object.
+   */
+  addendum: string
+  /**
+   * ID to find this component in testing tools (e.g.: Cypress, Testing Library, and Jest).
+   */
+  testId?: string
+}
+
+const NewsletterAddendum = forwardRef<HTMLSpanElement, NewsletterAddendumProps>(
+  function NewsletterAddendum(
+    { addendum, testId = 'fs-newsletter-addendum', ...otherProps },
+    ref
+  ) {
+    return (
+      <span
+        ref={ref}
+        data-testid={testId}
+        data-fs-newsletter-addendum
+        dangerouslySetInnerHTML={{ __html: addendum }}
```

Comment:
I know this reflects the default behavior from core, but it comes out a little weird to me that this component expects the HTML value by default.

That is true only for our use, and I don't see other people using HTML for this component inside custom sections. I think they'd put the value directly, and for that using `children` is a better API IMO.

With `...otherProps` `@faststore/core` would be able to set the HTML the way we do here.

I think in this specific case it's close to a nitpick but in general terms the question is: should we "leak" the pattern of usage from `@faststore/core` to `@faststore/ui`?